golang/testsuite/networks_impl/test_network.go

```diff
@@ -1,33 +1,45 @@
+/*
+ * Copyright (c) 2021 - present Kurtosis Technologies Inc.
+ * All Rights Reserved.
+ */
+
 package networks_impl
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/kurtosis-tech/example-microservice/api/api_service_client"
-	"github.com/kurtosis-tech/example-microservice/datastore/datastore_service_client"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/kurtosis-tech/example-api-server/api/golang/example_api_server_rpc_api_bindings"
+	"github.com/kurtosis-tech/example-datastore-server/api/golang/datastore_rpc_api_bindings"
 	"github.com/kurtosis-tech/kurtosis-client/golang/lib/networks"
 	"github.com/kurtosis-tech/kurtosis-client/golang/lib/services"
 	"github.com/palantir/stacktrace"
 	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/emptypb"
 	"io/ioutil"
 	"os"
 	"strconv"
+	"time"
 )
 
 const (
-	datastoreImage                        = "kurtosistech/example-microservices_datastore"
 	datastoreServiceId services.ServiceID = "datastore"
 	datastorePort                         = 1323
```

Comment:
Same comment as from https://github.com/kurtosis-tech/kurtosis-cli/pull/61 , in that I think you can now replace this with the actual datastore from the datatstore API lib